src/ReactiveUI.Tests/Utilities/ApiApprovalBase.cs

```diff
@@ -83,7 +83,7 @@ protected static void CheckApproval(Assembly assembly, [CallerMemberName]string?
                 }
             }
 
-            Assert.Equal(approvedPublicApi, receivedPublicApi);
+            Assert.Equal(approvedPublicApi.Trim(), receivedPublicApi);
```

Comment:
I don't know why my VS keeps adding a blank line at the end of the approval files when saving, but this fixes it.